wp-graphql.php

```diff
@@ -72,7 +72,7 @@ function graphql_init_appsero_telemetry() {
 		return;
 	}
 
-	$client   = new Appsero\Client( 'cd0d1172-95a0-4460-a36a-2c303807c9ef', 'WP GraphQL', __FILE__ );
+	$client   = new Appsero\Client( 'cd0d1172-95a0-4460-a36a-2c303807c9ef', 'WPGraphQL', __FILE__ );
```

Comment:
I'm not sure what the effects of changing the 2nd argument here would be, but wanted to update to discuss within this PR.

Reply:
@josephfusco, ironically Appsero actually already says this should be WPGraphQL without the space. . .so I'll merge this and monitor to make sure it doesn't impact Appsero tracking in a negative way. 

Thanks 🙌🏻